src/components/field-label/field-label.spec.js

```diff
@@ -114,6 +114,21 @@ describe('rendering', () => {
       it('should set the icon size', () => {
         expect(hintIconWrapper).toHaveProp('size', 'medium');
       });
+
+      describe('with custom theme', () => {
+        beforeEach(() => {
+          props = createTestProps({
+            hint: 'foo',
+            hintIcon: <WarningIcon theme="green" />,
+          });
+          wrapper = shallow(<FieldLabel {...props} />);
+          hintIconWrapper = wrapper.find(WarningIcon);
+        });
+
+        it('should set the icon theme', () => {
+          expect(hintIconWrapper).toHaveProp('theme', 'green');
```

Comment:
Just making sure the prop is forwarded so the change can't be reverted without a test failing.